modules/mapping-utils/src/resolveSets.js

```diff
@@ -67,17 +269,18 @@ export const saveSet = ({ types, callback }) => async (
   if (tag) {
     // if a tag is present, test early.
     if (!isTagValid(tag)) {
-      throw new Error('Invalid tag, no set created.');
+      return null;
     } else if (!isFunction(callback)) {
-      throw new Error('Cannot process further, no set created.');
+      return null;
```

Comment:
since return is the same. cant they be merged into the same if condition?

Reply:
good point